hathor/profiler/resources/cpu_profiler.py

```diff
@@ -0,0 +1,142 @@
+"""
+Copyright 2019 Hathor Labs
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
Just a few last nitpicks: this should be commented instead of a docstring (on the sec-dev branch we already have this change, not doing it here will lead to it being inconsistent when merged, instead of conflicting), I'd update the year (wow, it's 2021 already, haha), and lastly I'd add this to all new files. To be honest I don't know the extent of how much this is needed, but since we decided to add these headers I'd try to keep them just in case.

Reply:
Done! Thanks!